### PR TITLE
contact page | aos | fix css

### PR DESCRIPTION
**Pull request by KobiHay**
* contact page
* add animate on scroll (aos) for the homepage
* fix some of css design in homepage & footer

### CODE REVIEW

File: our-brothers-web/package.json [hunk 1]
```diff
@@ -27,8 +27,10 @@
     "@angular/platform-browser-dynamic": "~8.2.14",
     "@angular/router": "~8.2.14",
     "@fortawesome/fontawesome-free": "^5.12.1",
+    "aos": "^2.3.4",
     "firebase": "~7.6.0",
     "js-marker-clusterer": "~1.0.0",
+    "ng2-animate-on-scroll": "^1.2.0",
```
Comment (gilhanan) [inline our-brothers-web/package.json]: Do we need this?

File: our-brothers-web/package.json [hunk 2]
```diff
@@ -27,8 +27,10 @@
     "@angular/platform-browser-dynamic": "~8.2.14",
     "@angular/router": "~8.2.14",
     "@fortawesome/fontawesome-free": "^5.12.1",
+    "aos": "^2.3.4",
```
Comment (gilhanan) [inline our-brothers-web/package.json]: Please change to `~2.3.4` and also change `@fortawesome/fontawesome-free` to `~5.12.1`

File: our-brothers-web/src/app/pages/contact-page/contact-page.component.scss [hunk 3]
```diff
@@ -0,0 +1,127 @@
+.meetings-image-header {
```
Comment (gilhanan) [inline our-brothers-web/src/app/pages/contact-page/contact-page.component.scss]: Do we use this class?

File: our-brothers-web/src/app/pages/home-page/home-page.component.scss [hunk 4]
```diff
@@ -66,13 +66,18 @@
   justify-content: flex-end;
```
Comment (gilhanan) [inline our-brothers-web/src/app/pages/home-page/home-page.component.scss]: Please lint

File: our-brothers-web/src/app/components/footer/footer.component.scss [hunk 5]
```diff
@@ -138,6 +138,11 @@
         font-size: 11px;
       }
 
+      .publisher-link{
+        text-decoration: none;
+        color:#999999;
```
Comment (gilhanan) [inline our-brothers-web/src/app/components/footer/footer.component.scss]: Please lint

File: our-brothers-web/package.json (hunk 1 above)
Comment (KobiHay) [reply our-brothers-web/package.json]: fixed

File: our-brothers-web/package.json (hunk 2 above)
Comment (KobiHay) [reply our-brothers-web/package.json]: fixed

File: our-brothers-web/src/app/pages/contact-page/contact-page.component.scss (hunk 3 above)
Comment (KobiHay) [reply our-brothers-web/src/app/pages/contact-page/contact-page.component.scss]: yes

File: our-brothers-web/src/app/pages/home-page/home-page.component.scss (hunk 4 above)
Comment (KobiHay) [reply our-brothers-web/src/app/pages/home-page/home-page.component.scss]: fixed

File: our-brothers-web/src/app/components/footer/footer.component.scss (hunk 5 above)
Comment (KobiHay) [reply our-brothers-web/src/app/components/footer/footer.component.scss]: fixed

**Pull request closed without merging**